src/OrbitQt/orbittablemodel.cpp

```diff
@@ -57,7 +57,8 @@ QVariant OrbitTableModel::data(const QModelIndex& index, int role) const {
   if (role == Qt::DisplayRole) {
     std::string value = data_view_->GetValue(index.row(), index.column());
     return QVariant(QString::fromStdString(value));
-  } else if (role == Qt::ForegroundRole) {
+  }
+  if (role == Qt::ForegroundRole) {
```

Comment:
Hmm, I don't agree with it here: You cannot do the same with the following else, because this branch doesn't always return. So the if-elseif chain is split here, but the same cannot be done below. This makes the code more confusing.

Reply:
Well, actually you _can_ split the chain in the other places as well, because `if (role == Qt::ForegroundRole)` holds, then the other `if`s don't, so that's probably the way to go.